rts/Game/UI/KeySet.cpp

```diff
@@ -108,7 +108,7 @@ bool CKeySet::Parse(const std::string& token, bool showerror)
 
 	// parse the modifiers
 	while (!s.empty()) {
-		     if (ParseModifier(s, "up+",    "u+")) { modifiers |= KS_RELEASE; }
+		if      (ParseModifier(s, "up+",    "u+")) { modifiers |= KS_RELEASE; }
```

Comment:
I was mostly trying to get clang-tidy not to report [`readability-misleading-indentation`](https://clang.llvm.org/extra/clang-tidy/checks/readability-misleading-indentation.html) here, as it can be a pretty serious source of errors.

If you don't mind, I'd like to turn explicitly [suppress](http://clang.llvm.org/extra/clang-tidy/#suppressing-undesired-diagnostics) clang-tidy diagnostics in some similar cases where we have our unique style, and especially where following the clang-tidy instructions could lead to programmer [errors](https://github.com/spring/spring/commit/d86e671010e1c17e321d431dc9d6efd90c02963e)

Reply:
Hmm, how many `// NOLINT`'s would this add?

Reply:
Turned out to be too many in this particular case so I rewrote it as suggested. 
Some examples of it can now be seen at https://github.com/spring/spring/pull/414/files#diff-8b4fb662f60d9e914aa2279c5e6abe3eR90